src/code-components/RichTextArea/Editor.tsx

```diff
@@ -79,6 +119,19 @@ const Editor = forwardRef<Quill | null, EditorProps>(
         theme: "snow",
         modules: {
           toolbar: toolbarConfigs,
+          imageDropAndPaste: {
+            handler: async (_: string, type: string, data: QuillImageData) => {
+              const imageDataWithName = data as QuillImageData & {
+                name?: string;
+              };
+
+              const imageUrl = await imageDropHandler(type, imageDataWithName);
```

Comment:
This function is used only in this use effect, so it should be defined here, why would we construct it on every render?